plugins/woocommerce/client/legacy/css/admin.scss

```diff
@@ -5942,9 +5942,24 @@ img.help_tip {
 	.wc-settings-row-woocommerce_custom_orders_table_data_sync_enabled td.forminp {
 		padding-top: 0;
 	}
+
 	#email_notification_settings-description {
 		margin-top: 16px;
 	}
+
+	.learn-more-link {
```

Comment:
Non-blocking: we could increase the left margin of the link:

<img width="930" height="274" alt="Screenshot from 2025-09-25 08-57-26" src="https://github.com/user-attachments/assets/7700a8c7-682f-42c3-89b6-5eed2d00d70a" />